### PR TITLE
Setup Material UI

### PR DESCRIPTION
**Pull request by Yvad60**
Material UI is well configured using styled components

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,7 +1,14 @@
 import React from 'react';
+import { Button, Typography } from '@mui/material';
 
 function App() {
-	return <h1>Ivad Code</h1>;
+	return (
+		<div>
+			<Typography>Ivad code</Typography>
+
```
Comment (Abdulkeza) [inline src/App.jsx:8]: Reduce unused spaces... but no bigg

**Review by Abdulkeza — APPROVED** [review]
Great job Mr yvad..

File: src/App.jsx (hunk 1 above)
Comment (Yvad60) [reply src/App.jsx:8]: I think that's an issue with GitHub. let me confirm that prettier is working as expected

Comment (Yvad60) [reply src/App.jsx:8]: @Abdulkeza prettier is working as expected and my code are well-formatted locally definitely the issues with Github

**Pull request merged**